nautobot/extras/models/relationships.py

```diff
@@ -709,6 +713,10 @@ class Meta:
             "destination_type",
             "destination_id",
         )
+        index_together = [
```

Comment:
Mind adding a comment explaining the usefulness of these two indexes? They're not as self-explanatory as the ones above.

Reply:
Yeah right now it's actually looking like the indexes haven't made much of a difference and I may be wrong here.